man/man1/git-secret-hide.1.md

```diff
@@ -7,37 +7,29 @@ git-secret-hide - encrypts all added files with the inner keyring.
 
 
 ## DESCRIPTION
-`git-secret-hide` creates an encrypted version (typically called `filename.txt.secret`) 
-of each file added by `git-secret-add` command. 
+`git-secret-hide` creates an encrypted version (typically called `filename.txt.secret`)
+of each file added by `git-secret-add` command.
 Now anyone enabled via 'git secret tell' can can decrypt these files. Under the hood,
 `git-secret` uses the keyring in `.gitsecret/keys` and user's secret keys to decrypt the files.
 
-It is recommended to encrypt (or re-encrypt) all the files in a git-secret repo each 
+It is recommended to encrypt (or re-encrypt) all the files in a git-secret repo each
 time `git secret hide` is run.
 
 Otherwise the keychain (the one stored in `.gitsecret/keys/*.gpg`),
-may have changed since the last time the files were encrypted, and it's possible 
-to create a state where the users in the output of `git secret whoknows` 
-may not be able to decrypt the some files in the repo, or may be able decrypt files 
+may have changed since the last time the files were encrypted, and it's possible
+to create a state where the users in the output of `git secret whoknows`
+may not be able to decrypt the some files in the repo, or may be able decrypt files
 they're not supposed to be able to.
 
-In other words, unless you re-encrypt all the files in a repo each time you 'hide' any, 
-it's possible to make it so some files can no longer be decrypted by users who should be 
+In other words, unless you re-encrypt all the files in a repo each time you 'hide' any,
+it's possible to make it so some files can no longer be decrypted by users who should be
 (and would appear) able to decrypt them, and vice-versa.
 
-If you know what you are doing and wish to encrypt or re-encrypt only a subset of the files 
-even after reading the above paragraphs, you can use the -F or -m option to only encrypted 
-a subset of files. The -F option forces `git secret hide` to skip any hidden files 
-where the unencrypted versions aren't present. The -m option skips any hidden files that have 
-not be modified since the last time they were encrypted. 
-
-Also, it is possible to modify the names of the encrypted files by setting `SECRETS_EXTENSION` variable.
-
-(See [git-secret(7)](http://git-secret.io/git-secret) for information about renaming the .gitsecret
-folder using the SECRETS_DIR environment variable.
-
-You can also enable verbosity using the SECRETS_VERBOSE environment variable,
-as documented at [git-secret(7)](http://git-secret.io/)
+If you know what you are doing and wish to encrypt or re-encrypt only a subset of the files
+even after reading the above paragraphs, you can use the `-F` or `-m` option to only encrypted
+a subset of files. The `-F` option forces `git secret hide` to skip any hidden files
```

Comment:
we should remove "to only encrypted (sic) a subset of files." from the end of the sentence starting with "If you know what you are doing", and replace it with "to do so", because it repeats what we say in start of sentence.